Sources/Afterpay/Views/PriceBreakdownView.swift

```diff
@@ -28,17 +28,9 @@ public final class PriceBreakdownView: UIView {
   private var linkColor: UIColor!
   private let colorScheme: ColorScheme
 
-  private var termsLink: String {
-    switch getLocale() {
-    case Locales.australia:
-      return "https://static-us.afterpay.com/javascript/modal/au_rebrand_modal.html"
-    case Locales.newZealand:
-      return "https://static-us.afterpay.com/javascript/modal/nz_rebrand_modal.html"
-    case Locales.canada:
-      return "https://static-us.afterpay.com/javascript/modal/ca_rebrand_modal.html"
-    default:
-      return "https://static-us.afterpay.com/javascript/modal/us_rebrand_modal.html"
-    }
+  private var infoLink: String {
+    let region = (getLocale().regionCode ?? "US").lowercased()
```

Comment:
I wonder if we can avoid a hardcoded region code here?

```suggestion
    let locale = getLocale() ?? Locales.unitedStates
    let region = locale.regionCode.lowercased()
```

Reply:
Sadly `regionCode` is defined as `String?` I thought about making a region wrapper of some kind?

Reply:
`getLocale()` actually returns a non optional `Locale`

Reply:
Ah my bad, I should have picked that up. It's fine as is—I don't think there's any need for a wrapper right now